<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatePicker</class>
 <widget class="QWidget" name="DatePicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick dates</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="startLbl">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startDatePicker">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="endLbl">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>End date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endDatePicker">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="cbMon">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mon</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbTue">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tue</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbWed">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wed</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbThu">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thu</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbFri">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fri</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbSat">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbSun">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sun</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
